deployment/agent-deployment/ServiceGroupRoot/Parameters/ContainerInsights.Linux.Parameters.json

```diff
@@ -0,0 +1,84 @@
+{
+    "$schema": "http://schema.express.azure.com/schemas/2015-01-01-alpha/RolloutParameters.json",
+    "contentVersion": "1.0.0.0",
+    "wait": [
+      {
+        "name": "waitSdpBakeTime",
+        "properties": {
+          "duration": "PT24H"
+        }
+      }
+    ],
+    "shellExtensions": [
+      {
+        "name": "PushAgentToACR",
+        "type": "ShellExtensionType",
+        "properties": {
+          "maxexecutiontime": "PT1H"
+        },
+        "package": {
+          "reference": {
+            "path": "artifacts.tar.gz"
+          }
+        },
+        "launch": {
+          "command": [
+            "/bin/bash",
+            "pushAgentToAcr.sh"
+          ],
+          "environmentVariables": [
+            {
```

Comment:
I think, we can remove this "WINDOWS" environment variable from here, right? For the Windows ( in windows parameter file), you will be having this environment variable with value "win-". if the value of "WINDOWS" either empty or doesnt exist, determines that, its Linux, right?

Reply:
Yes--removing now